Tools/OSD/Panels.cs

```diff
@@ -917,14 +926,33 @@ public int panGPSats(int first_col, int first_line, int sign, int fAlt)
             osd.setPanel(first_col, first_line);
             
             if(sign==1)
-            	osd.printf("%c%2i", 0x2a, osd_satellites_visible);
+            	osd.printf("%2i%c", osd_satellites_visible, 0x86);

```

Comment:
really stupid, why not osd.printf("%2i\x86", osd_satellites_visible) ?? I will not tell about special cheap functions...